File: 123_2.java
import java.util.ArrayList;
import java.util.List;

public class Main {
    public static void main(String[] args) {
        List<String> li = new ArrayList<>();
        li.add("a");
        li.add("1");
        li.add("b");
        li.add("2");
        li.add("c");
        li.add("3");
        
        List<String> li_1 = new ArrayList<>(); // для букв
        List<String> li_2 = new ArrayList<>(); // для чисел
        
        // Разделяем список на буквы и числа
        for (int i = 0; i < li.size(); i++) {
            if (i % 2 == 0) {
                li_1.add(li.get(i)); // четные индексы - буквы
            } else {
                li_2.add(li.get(i)); // нечетные индексы - числа
            }
        }

File: 123_2.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {
    vector<string> li = {"a", "1", "b", "2", "c", "3"};
    vector<string> li_1; // для букв
    vector<string> li_2; // для чисел
    
    // Разделяем список на буквы и числа
    for (int i = 0; i < li.size(); i++) {
        if (i % 2 == 0) {
            li_1.push_back(li[i]); // четные индексы - буквы
        } else {
            li_2.push_back(li[i]); // нечетные индексы - числа
        }
    }
